### PR TITLE
Add ability to send command with UnregisteredContactMessage

### PR DESCRIPTION
**Pull request by temagi**

### CODE REVIEW

File: examples/echo_bot.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 from pyAitu import Bot, Dispatcher, executor
 from pyAitu.models import Message, QuickButtonSelected, InlineCommandSelected, ContentType,\
     QuickButtonCommand, InlineCommand, ReplyCommand, Media
+from pyAitu.models.media.contact import Contact
```
Comment (yerassyl94) [inline examples/echo_bot.py]: please import only from models, it means define Contact in media.__init__ and models.__init__

File: pyAitu/bot/bot.py [hunk 2]
```diff
@@ -1,7 +1,8 @@
 from typing import List, Dict, Optional
 from .base import BaseBot
 from ..models import Update, Media, Command, QuickButtonCommand, InlineCommand, ReplyCommand, Form
-from ..utils.strings import COMMANDS, SEND_MESSAGE, GET_UPDATES, UPLOADED_FILES, SEND_UI_STATE
+from ..models.media.contact import Contact
```
Comment (yerassyl94) [inline pyAitu/bot/bot.py]: same issue above

File: pyAitu/models/command/command.py [hunk 3]
```diff
@@ -27,10 +27,13 @@ def create_command(
...
             reply_keyboard: list = None,
             quick_button_commands: list = None,
-            form: Form = None
+            form: Form = None,
+            input_media: Contact = None
     ):
         commands = []
         form_message = FormMessage(form).__dict__ if form is not None else {}
+        if input_media is not None:
+            input_media = input_media.to_dict()
```
Comment (yerassyl94) [inline pyAitu/models/command/command.py]:
```suggestion
            input_media = input_media.to__dict__
```

File: pyAitu/models/media/contact.py [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+
+class Contact():
+    def __init__(self, first_name: str = None, last_name: str = None, phone_number: str = None):
+        self.first_name = first_name
+        self.last_name = last_name
+        self.phone_number = phone_number
+        self.contact_type = "InputUnregisteredContact"
+
+    def to_dict(self):
```
Comment (yerassyl94) [inline pyAitu/models/media/contact.py]: creation of this function is redundant, please remove.

File: pyAitu/utils/strings.py [hunk 5]
```diff
@@ -24,3 +25,8 @@
 SHOW_SHARE_CONTACT_BUTTON = "showShareContactButton"
 SHOW_RECORD_AUDIO_BUTTON = "showRecordAudioButton"
 SHOW_GALLERY_BUTTON = "showGalleryButton"
+INPUT_MEDIA = "inputMedia"
+CONTACT_TYPE = "contactType"
+FIRST_NAME = "firstName"
+LAST_NAME = "lastName"
+PHONE_NUMBER = "phoneNumber"
```
Comment (yerassyl94) [inline pyAitu/utils/strings.py]: I guess this strings wont be required

File: pyAitu/models/media/contact.py [hunk 6]
```diff
@@ -0,0 +1,17 @@
+from pyAitu.utils.strings import CONTACT_TYPE, FIRST_NAME, LAST_NAME, PHONE_NUMBER
+
+
+class Contact():
+    def __init__(self, first_name: str = None, last_name: str = None, phone_number: str = None):
+        self.first_name = first_name
+        self.last_name = last_name
+        self.phone_number = phone_number
+        self.contact_type = "InputUnregisteredContact"
```
Comment (yerassyl94) [inline pyAitu/models/media/contact.py]: make sure that variable names are correct

**Review by yerassyl94 — CHANGES_REQUESTED** [review]

File: examples/echo_bot.py (hunk 1 above)
Comment (temagi) [reply examples/echo_bot.py]: fixed

File: pyAitu/bot/bot.py (hunk 2 above)
Comment (temagi) [reply pyAitu/bot/bot.py]: fixed

File: pyAitu/models/command/command.py (hunk 3 above)
Comment (temagi) [reply pyAitu/models/command/command.py]: fixed

File: pyAitu/utils/strings.py (hunk 5 above)
Comment (temagi) [reply pyAitu/utils/strings.py]: fixed

File: pyAitu/models/media/contact.py (hunk 6 above)
Comment (temagi) [reply pyAitu/models/media/contact.py]: fixed

File: pyAitu/models/media/contact.py (hunk 4 above)
Comment (temagi) [reply pyAitu/models/media/contact.py]: fixed

**Review by yerassyl94 — APPROVED** [review]

**Pull request merged**